### DISCUSSION TITLE
MaxCommandChainLength set in config.yml

### DISCUSSION THREAD

**Discussion by Cryper** [Zrips/CMI#3621] [Feature Requests]
**Description of feature/improvement request:**
It would be great if CMI will get a option to set the "maxCommandChainLength" instead of override it. I set it via gamerule to 1024. But i cant do more than 256.
So its impossible to execute a command thats longer, even with the console its not possible
---

**Comment by CloudeLecaw** [comment]
And you can execute vanilla command now with the. Lenght of 1024¿

**Comment by Cryper** [comment]
I can execute the vanilla command , but it dont change anything. If i set it to 1028 , the command cant be longer than 256.

**Comment by CloudeLecaw** [comment]
your interepreting thsi command gamerule also wrong way

What your searching for is something like a longer chat feature for executing longer chats or commands in one line. this needs an ourput"cache" that saves the 256 (+256+256+256+.....=until the letteramount is not equal 256 anymore and then print all out at once.

i had something like this once with a chatplugin and you where able this way to make long chats over 20000 words at once and also executing extream long commands with this

but back to your gamerule - why i think you interpreted it wrong:
This is what the wiki tells us:
> maxCommandChainLength | Determines the number at which the chain command block acts as a "chain". | 65536 | Int | Yes | Yes
> -- | -- | -- | -- | -- | --

This means u shorted the value allready that is by default  allready 65536, so if your maximum allowed chaisn in the commandblocks are 1024 (the value u set it too) It changes nothing to the Chat or console input you can do at once.

This also just works for commandblock chains not for your normal comamnds from char or console.
can work like this -> "If playerLvl2 then kill Player else heal player"  this is a chain that is meant with this gamerule
if you set it to 0 or negative then no chains can be executed anymore from commandblocks then just a normal command can be executed at once even if the chain is longer

**Comment by Cryper** [comment]
I tried the long command on a vanilla spigot server. First i tried it with maxCommandChainLength 256 . And the command dont work, cause it was to long.
After that i changed maxCommandChainLength to 1028. And the command worked.  

I know that this is made for Commandblock etc. but if i enable CMI , the command block stops at 256 too. So CMI do override this
